### ISSUE TITLE
15 'is about' subproperty definitions. 

### ISSUE THREAD

**Issue by neilotte** [CommonCoreOntology/CommonCoreOntologies#103]
For 15 sub relations of 'is about' within CCO, the beginnings of their definitions are formulated with the clause: 

"For all Types T1 and T2, if T1 **RELATION** T2 then..."

Take the definition of designates: "For all types T1 and T2, if T1 designates T2, then there is some instance of T1, t1, by which some instance of T2, t2,  is called, or classified, distinguished from others." 

What is T1 such that one could designate the T2 AND have instances? Since 'Types' is undefined, I think it's natural to read 'type' as 'class'. But then these relations would hold between classes and not instances. I'm really lost in these formulations. 

Is there a reason this couldn't be simplified to something like the following: 
'If x designates y, then x is an information content entity and y is called, or classified, or referred to by x.'

`SELECT ?x
WHERE
{
?x cco:definition ?o
FILTER (regex(str(?o), "For all types T1 and T2, if T1", "i"))
}`

**Comment by mark-jensen** [comment]
@neilotte Which version of CCO are you working with? Many relations were updated as part of the last [release](https://github.com/CommonCoreOntology/CommonCoreOntologies/releases/tag/v1.3). Latest version IRIs should contain `2020-09-23`.

We updated many of the property definitions to something close to what you're suggesting. E.g:

"x designates y iff x is an instance of an Information Content Entity, and y is an instance of an Entity, such that given some context, x uniquely distinguishes y from other entities"
